### DISCUSSION TITLE
Anyone Who Might Want To Provide Tutoring or Lessons Online

### DISCUSSION THREAD

**Discussion by ingmasis** [Ifsttar/I-Simpa#340] [General]
I emailed the University and I thought I will put up post here to see if anyone or any previous students or even current students might want to teach I-Simpa online one on one. And if there is any University in a developing world that teaches I-Simpa that might have students who would offer this at an affordable price point that would be great. In any case it seems that it is a French Uni so doesn't really matter if anyone might offer this please let me know.

I am a not an acoustics student and unfortnately dont have any CAD experience but fully understand acoustics concepts to understand how to and the purpose of I-Simpa.

I am finding it diffucult to teach myslelf.

**Comment by sidway** [comment]
Hi, i think i just reply your comment on a random video on YouTube. Yes, this software was developed on [Université Gustave Eiffel](https://www.univ-gustave-eiffel.fr/). I think nicolas-f is the main contributor today. 

I use i-Simpa on a daily basis, I can help you out. E-mail me your questions/needs so I can decide how we do this. I recommend that you try to learn Sketch up, is very intuitive, and you can export the room geometry to i-Simpa.

My e-mail is sidcandido.acustica@gmail.com.
